data/sheragi/sheragi outfits.txt

```diff
@@ -263,7 +263,7 @@ outfit "Particle Waveform Turret"
 		"fire effect" "lightning flare"
 		"hit effect" "pwave impact"
 		"inaccuracy" 1.5
-		"turret turn" 1.8
+		"turret turn" 0.8
```

Comment:
Could you explain why you feel this was nerfed too much? 
Is there a specific point of comparison you're using where you feel that the Particle Waveform Turret is now too slow relative to another turret with its rebalanced turn rate?

Reply:
If you look at the **Rebalance Philosophy** section above in the PR description, you'll see the criteria I used for determining rebalanced turret rates, specifically:

> Currently, I am basing the rebalanced turn rates primarily on the size and range of the turret, with damage, purpose, and overall tech level being lesser considerations.
Size is internally consistent if, all else being equal, the larger a turret is, the slower it can turn.
Range, as the range of a turret is intrinsically linked to turning. The more range a turret has, the less turning it needs to be able to effectively engage targets at its maximum range.
Damage isn't as much of a consideration as the damage a turret can do isn't directly related to its ability to target effectively. Damage is also generally linked to size, so it's considered that way.

So, under the rebalancing philosophy I'm using, 
The larger a turret is, the slower it turns
The longer-ranged a turret is, the slower it turns
The more damage a turret does, the slower it turns - to a lesser extent
The lower-tech a turret is, the slower it turns - to a lesser extent


With those criteria in mind, compare the Particle Waveform Turret to the Pulse Turret -
27% Larger
100% More Range
29% Less Damage
+420/s Ion Damage
+224/s Heat Damage
Lower Tier

Larger, longer-ranged, and lower tier all mean it should turn slower. 
The addition of ion and heat damage offsets the lower DPS.

Considering all that, the 46% slower turning rate doesn't seem unreasonable. As far as I can tell, it is consistent with my rebalancing philosophy.